snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/dsv/SnomedSimpleTypeRefSetDSVExporter.java

```diff
@@ -34,44 +37,87 @@
 import com.b2international.snowowl.snomed.common.SnomedRf2Headers;
 import com.b2international.snowowl.snomed.core.domain.SnomedConcept;
 import com.b2international.snowowl.snomed.core.domain.SnomedConcepts;
-import com.b2international.snowowl.snomed.core.domain.SnomedDescription;
-import com.b2international.snowowl.snomed.core.domain.SnomedRelationship;
-import com.b2international.snowowl.snomed.core.domain.refset.SnomedRefSetType;
 import com.b2international.snowowl.snomed.datastore.index.entry.SnomedConceptDocument;
 import com.b2international.snowowl.snomed.datastore.internal.rf2.AbstractSnomedDsvExportItem;
 import com.b2international.snowowl.snomed.datastore.internal.rf2.ComponentIdSnomedDsvExportItem;
 import com.b2international.snowowl.snomed.datastore.internal.rf2.DatatypeSnomedDsvExportItem;
 import com.b2international.snowowl.snomed.datastore.internal.rf2.SnomedRefSetDSVExportModel;
 import com.b2international.snowowl.snomed.datastore.request.SnomedConceptSearchRequestBuilder;
 import com.b2international.snowowl.snomed.datastore.request.SnomedRequests;
+import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.Charsets;
 import com.google.common.base.Joiner;
-import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.ImmutableSortedSet;
-import com.google.common.collect.Ordering;
+import com.google.common.collect.*;
 
 /**
  * Implements the export process of the DSV export for simple type reference sets. 
  */
 public class SnomedSimpleTypeRefSetDSVExporter implements IRefSetDSVExporter {
 
-	private static final String HEADER_EXPAND = "descriptions(active:true),"
-			+ "relationships(active:true),"
-			+ "members()";
+	private static final String HEADER_EXPAND = "descriptions(active:true), "
+			+ "relationships(active:true), "
+			+ "members(active:true, refSetType:\"CONCRETE_DATA_TYPE\")";
 	
-	private static final String DATA_EXPAND = "pt(),"
-			+ "descriptions(active:true),"
-			+ "relationships(active:true,expand(destination(expand(pt())))),"
-			+ "members()";
+	private static final String DATA_EXPAND = "pt(), "
+			+ "descriptions(active:true), "
+			+ "relationships(active:true, expand(destination(expand(pt())))), "
+			+ "members(active:true, refSetType:\"CONCRETE_DATA_TYPE\")";
 
-	private static final Map<String, Integer> NO_OCCURRENCES = ImmutableMap.of();
+	private static final Multiset<String> NO_OCCURRENCES = ImmutableMultiset.of();
+	
+	static interface ConceptStreamFactory {
+		
+		Stream<SnomedConcepts> getConceptStream(
+			String expand, 
+			List<ExtendedLocale> locales, 
+			BranchContext context, 
+			boolean includeInactiveMembers,
+			String refSetId
+		);
+		
+		ConceptStreamFactory DEFAULT = (expand, locales, context, includeInactiveMembers, refSetId) -> {
+			SnomedConceptSearchRequestBuilder builder = SnomedRequests.prepareSearchConcept()
+				.setLocales(locales)
+				.setExpand(expand)
+				.sortBy(Sort.fieldAsc(SnomedConceptDocument.Fields.ID))
```

Comment:
Btw, this is the default sort field for all revision documents.

Reply:
I see a `_default` sort here: https://github.com/b2ihealthcare/snow-owl/blob/5f2240d086432d66d32a8e441e062e8f7937ae43/commons/com.b2international.index/src/com/b2international/index/query/SortBy.java#L47-L55

Does the mapping mark `id` as the default field?

Reply:
Yes, for Revision documents that's the default sort field: 
https://github.com/b2ihealthcare/snow-owl/blob/9.x/commons/com.b2international.index/src/com/b2international/index/revision/Revision.java#L106-L107

Reply:
Sorting removed in https://github.com/b2ihealthcare/snow-owl/pull/1349/commits/8701276e51aa71747ad40a4aa3700659151f40a3.